power_grid_model_c/power_grid_model/include/power_grid_model/math_solver/math_solver.hpp

```diff
@@ -171,6 +173,9 @@ template <bool sym> class MathSolver {
 
 template class MathSolver<true>;
 template class MathSolver<false>;
+} // namespace math_solver
+
+template <bool sym> using MathSolver = math_solver::MathSolver<sym>;
```

Comment:
please double-check but i think that just doing

```suggestion
using math_solver::MathSolver;
```

may also work...

Reply:
It does work